.github/workflows/ci.yml

```diff
@@ -125,14 +125,14 @@ jobs:
           sentry_project_uat = os.environ['SENTRY_PROJECT_UAT']
           sentry_dns_uat = os.environ['SENTRY_DNS_UAT']
           if branch == staging:
-            print('::set-env name=SENTRY_PROJECT::' + sentry_project_staging)
-            print('::set-env name=SENTRY_DNS::' + sentry_dns_staging)
+            echo "SENTRY_PROJECT=sentry_project_staging" >> $GITHUB_ENV
+            echo "SENTRY_DNS=sentry_dns_staging" >> $GITHUB_ENV
           elif branch == production:
-            print('::set-env name=SENTRY_PROJECT::' + sentry_project_production)
-            print('::set-env name=SENTRY_DNS::' + sentry_dns_production)
+            echo "SENTRY_PROJECT=sentry_project_production" >> $GITHUB_ENV
+            echo "SENTRY_DNS=sentry_dns_production" >> $GITHUB_ENV
           elif branch == uat:
-            print('::set-env name=SENTRY_PROJECT::' + sentry_project_uat)
-            print('::set-env name=SENTRY_DNS::' + sentry_dns_uat)
+            echo "SENTRY_PROJECT=sentry_project_uat" >> $GITHUB_ENV
+            echo "SENTRY_DNS=sentry_dns_uat" >> $GITHUB_ENV
```

Comment:
this is python, might still need the print statement

Reply:
also after reading the vulnerability report, i dont think we are logging any untrusted data here since these are all predefined by us

Reply:
thanks for the catch.. backing this out now.